src/pages/index.js

```diff
@@ -1,68 +1,53 @@
 import "./index.css";
 
-/*import avatarSrc from "../images/avatar.jpg";
-const avatarImage = document.getElementById("example-avatar");
-avatarImage.src = avatarSrc;
-
-import logoSrc from "../images/logo.svg";
-const logoImage = document.getElementById("logo");
-logoImage.src = logoSrc;
-
-import pencilSrc from "../images/pencil.svg";
-const pencilImage = document.getElementById("pencil");
-pencilImage.src = pencilSrc;
-
-import plusSrc from "../images/plus.svg";
-const plusImage = document.getElementById("plus");
-plusImage.src = plusSrc;
-
-import trashSrc from "../images/trash.svg";
-const trashImage = document.getElementById("trash");
-trashImage.src = trashSrc;
-
-console.log(trashImage.src);*/
+import { setButtonText } from "../utils/helpers.js";
 
 import {
   enableValidation,
   settings,
   resetValidation,
+  disableButton,
 } from "../scripts/validation.js";
 
-const initialCards = [
-  {
-    name: "Val Thorens",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-  {
-    name: "Restaurant terrace",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
-  },
-  {
-    name: "An outdoor cafe",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
-  },
-  {
-    name: "A very long bridge, over the forest and through the trees",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
-  },
-  {
-    name: "Tunnel with morning light",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
-  },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-  {
-    name: "Golden Gate bridge",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
+import Api from "../utils/Api.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "860f8737-013c-4470-9264-4d503187c9c3",
+    "Content-Type": "application/json",
   },
-];
+});
+
+api.getAppInfo().then(([cards]) => {
+  cards.forEach((card) => {
+    const cardElement = getCardElement(card);
+    cardsList.prepend(cardElement);
+  });
+
+  api
+    .getUserInfo()
```

Comment:
`.getUserInfo()` is not needed because you get the user info in `api.getAppInfo()` in line 22